Source/Modeling/Plugins/JoistAreaFeatures/CreationFeature.cs

```diff
@@ -201,7 +201,7 @@ private void Guideline_OnPickingUndone(object sender, EventArgs eventArgs)
         private void Guideline_OnPreviewRequested(object sender, ToleratedObjectEventArgs eventArgs)
         {
             Graphics.Clear();
-            var currentAppliedValues = GetAppliedAttributes(Component);
+            //var currentAppliedValues = GetAppliedAttributes(Component); 
 
```

Comment:
Add direct call to Component to fetch dialog data

Reply:
the variable is not used elsewhere